tasks/pkg-debian.yml

```diff
@@ -31,21 +36,29 @@
   with_items:
     - "deb http://apt.datadoghq.com/ stable main"
     - "deb http://apt.datadoghq.com/ stable 6"
+    - "deb http://apt.datadoghq.com/ stable 7"
   when: not ansible_check_mode
 
 - name: Ensure Datadog repository is up-to-date
   apt_repository:
-    repo: "{{ datadog_apt_repo }}"
-    state: "{% if datadog_agent5 %}absent{% else %}present{% endif %}"
+    filename: datadog_agent
```

Comment:
I think we should prefix/suffix the filename by `ansible` to avoid clash with any user created file. WDYT ?

This would also allow us to detect who dropped that repo file when troubleshooting user install.